cli/ssh.go

```diff
@@ -35,6 +36,8 @@ import (
 var workspacePollInterval = time.Minute
 var autostopNotifyCountdown = []time.Duration{30 * time.Minute}
 
+var errConnectionDropped = xerrors.New("SSH connection ended unexpectedly")
```

Comment:
I think we could inline this for now until it needs to be used elsewhere!

Reply:
Another point for inlining: it produces a stacktrace, otherwise we have to wrap the error to get a stack trace.